tests/unit/test_noise_form.py

```diff
@@ -168,7 +168,7 @@ def test_correct_forms_are_used(func, form, mocker):
     """Test that each interface noise function uses the correct form"""
     if func == "todo":
         pytest.skip(reason=f"TODO: implement function for {form.value} form")
-    mock = mocker.patch("pseudopeople.interface.noise_form")
+    mock = mocker.patch("pseudopeople.interface._generate_form")
```

Comment:
I don't think this should be changed. We shouldn't mock out `_generate_form` since we don't have any test coverage for it.

Reply:
It doesn't work otherwise. I didn't get far in tracking down why, but I can revisit.

Reply:
Resolved with new commit.